README.md

```diff
@@ -6,61 +6,199 @@ NOTE: This repository is a WORK IN PROGRESS.
+
+```python
+get_and_save_flusight_submission(
+    file_save_path=os.path.join(os.getcwd(), "example_flusight_submission.csv"),
 )
 ```
 
 
-## NHSN COVID Hospital Admissions
+## NHSN COVID And Flu Hospital Admissions
+
+Hospital admissions data for fitting from NHSN for COVID and Flu is included in `forecasttools` as well, for user experimentation. This data is current as of `2024-04-27` and comes from the website [HealthData.gov COVID-19 Reported Patient Impact and Hospital Capacity by State Timeseries](https://healthdata.gov/Hospital/COVID-19-Reported-Patient-Impact-and-Hospital-Capa/g62h-syeh). For influenza, the `previous_day_admission_influenza_confirmed` column is retained and for COVID the `previous_day_admission_adult_covid_confirmed` column is retained. As can be seen in the example below, some early dates for each jurisdiction do not have data.
+
+Shape: (81_713, 3)
+
+| state | date       | hosp |
+| ---   | ---        | ---  |
+| str   | str        | str  |
+|-------|------------|------|
+| AK    | 2020-03-23 | null |
+| AK    | 2020-03-24 | null |
+| AK    | 2020-03-25 | null |
+| AK    | 2020-03-26 | null |
+| AK    | 2020-03-27 | null |
+| AK    | 2020-03-28 | null |
+| AK    | 2020-03-29 | null |
+| AK    | 2020-03-30 | null |
+| …    | …         | …   |
+| WY    | 2024-04-21 | 0    |
+| WY    | 2024-04-22 | 2    |
+| WY    | 2024-04-23 | 1    |
+| WY    | 2024-04-24 | 1    |
+| WY    | 2024-04-25 | 0    |
+| WY    | 2024-04-26 | 0    |
+| WY    | 2024-04-27 | 0    |
+
+
+The fitting data is stored in `forecasttools` as a `polars` dataframe and is accessed via:
 
 ```python
+import forecasttools
+
+
+# access COVID data
+covid_nhsn_data = forecasttools.nhsn_hosp_COVID
+
+# access flu data
+flu_nhsn_data = forecasttools.nhsn_hosp_flu
+```
+
+The data was created by placing a csv file called `NHSN_RAW_20240926.csv` (the full NHSN dataset) into `./forecasttools` and running, in `data.py`, the following:
+
+```python
+# generate COVID dataset
 make_nshn_fitting_dataset(
     dataset="COVID",
     nhsn_dataset_path="NHSN_RAW_20240926.csv",
-    save_directory=os.getcwd(),
-    file_save_name="nhsn_hosp_COVID.csv"
+    file_save_path=os.path.join(os.getcwd(),"nhsn_hosp_COVID.csv")
 )
-```
-
-## NHSN Influenza Hospital Admissions
-
 
-```python
+# generate flu dataset
 make_nshn_fitting_dataset(
     dataset="flu",
     nhsn_dataset_path="NHSN_RAW_20240926.csv",
-    save_directory=os.getcwd(),
-    file_save_name="nhsn_hosp_flu.csv"
+    file_save_path=os.path.join(os.getcwd(),"nhsn_hosp_flu.csv")
 )
 ```
 
 ## Influenza Hospitalizations Forecast
 
+An example forecast stored in an Arviz `InferenceData` object is included for vignettes and user experimentation. This 28 day forecast for Texas was made using a spline regression model fitted to NHSN influenza data between 2022-08-08 and 2022-12-08. The `idata` object which includes the observed data and posterior predictive samples is given below:
+
+```
+Inference data with groups:
```

Comment:
This especially would benefit from quarto generation, as it would interactive.

Reply:
See #19